src/diagnostics/riscv64/disasm-riscv64.cc

```diff
@@ -79,6 +79,19 @@ class Decoder {
   void PrintImm20J(Instruction* instr);
   void PrintShamt(Instruction* instr);
   void PrintShamt32(Instruction* instr);
+  void PrintRvcImm(Instruction* instr);
+  void PrintRvcImmU(Instruction* instr);
+  void PrintRvcImmLdsp(Instruction* instr);
+  void PrintRvcImmLwsp(Instruction* instr);
+  void PrintRvcImmLd(Instruction* instr);
+  void PrintRvcImmLw(Instruction* instr);
+  void PrintRvcImmSdsp(Instruction* instr);
+  void PrintRvcImmSwsp(Instruction* instr);
+  void PrintRvcImmAddi4spn(Instruction* instr);
+  void PrintRvcImmAddi16sp(Instruction* instr);
+  void PrintRvcImmB(Instruction* instr);
+  void PrintRvcImmJ(Instruction* instr);
+  void PrintRvcShamt(Instruction* instr);
```

Comment:
Add number of bits in the immediate field to the `Print` function name to be consistent w/ other Print functions